lte/gateway/c/core/oai/tasks/gtpv1-u/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ set(GTPV1U_SRC
     gtpv1u_task.c
     gtp_tunnel_openflow.c
     gtp_tunnel_upf.c
+    ${PROJECT_SOURCE_DIR}/test/mock_tasks/gtp_tunnel_mock.c
```

Comment:
we should also be able to remove this from here and instead update the cmake files in the test directory.